src/orga/subteams.rst

```diff
@@ -2,12 +2,53 @@ A list of current sub-teams
 ***************************
 
 
+Security and Systems Sub-Team
+=============================
+
+Role
+----
+The purpose of the security and systems sub-team is to secure and maintain appropriate access
+to the credentials and services/systems used by ``conda-forge``. This infrastructure
+includes all bot accounts, all service provider accounts, and all keys, API or otherwise,
+used for various tasks. This team is also charged with provisioning new members of ``core``
+with access to the organization.
+
+Charter
+-------
+Static
+
+Responsibility
+--------------
+The core responsibilities of this team are to
+
+- maintain secure access to the credentials to all bot accounts
+- maintain secure access to all keys, API or otherwise, used by ``conda-forge``
+- maintain secure access to all service provider accounts owned by ``conda-forge``
+- maintain the infrastructure for CFEP-13
+- maintain automated ways to recover and reprovision ``conda-forge`` systems
+- perform ongoing security maintenance tasks
+- provision new members of `core` with access to all ``conda-forge`` systems, credentials
+  and keys
+
+This team will operate via the following rules
+
+- only members of `core` can be on this sub-team
+- this sub-team will report its activity to core at all ``core`` meetings
+- this sub-team will consult with core at ``core`` meetings regarding all planned activity
+- this sub-team will not limit the access by ``core`` members to any ``conda-forge`` resources
+  including but not limited to its systems, credentials, keys, or service accounts
+
+Members
+-------
+- Matthew R. Becker <becker.mr@gmail.com>
```

Comment:
@conda-forge/core This PR establishes the security and systems subteam per CFEP-14. If you'd like to join, please add your name here!

Reply:
It's gonna be lonely if it is just me! Anyone?

Reply:
I am happy to be on this team too.